src/encoded/audit/experiment.py

```diff
@@ -1373,11 +1395,20 @@ def check_experiment_cage_rampage_standards(experiment,
                               'MadQualityMetric',
                               desired_assembly,
                               desired_annotation)
+    replicates = experiment.get('replicates')
 
-    yield from check_spearman(
-        mad_metrics, experiment['replication_type'],
-        0.9, 0.8, 'RAMPAGE (paired-end, stranded)')
-    return
+    if experiment['replication_type'] == 'unreplicated' and len(replicates) > 1:
+        yield from check_spearman_technical_replicates(
+            mad_metrics, 'RAMPAGE (paired-end, stranded)', 0.9)
+
+        return
+
+    else:
+        yield from check_spearman(
+            mad_metrics, experiment['replication_type'],
+            0.9, 0.8, 'RAMPAGE (paired-end, stranded)')
+
+        return
```

Comment:
I think the technical MAD should be limited to bulk RNA only. 
we are not planning to run and we don't have a new RAMPAGE pipeline.